fahrplan/main.py

```diff
@@ -103,6 +107,9 @@ def assert_enough_arguments(args):
             logging.basicConfig(level=logging.INFO)
         if tokens[0] in ['-d', '--debug']:
             logging.basicConfig(level=logging.DEBUG)
+        if tokens[0] in ['-p', '--proxy']:
+            proxy_host = tokens[1]
+            del tokens[0]
```

Comment:
This parsing method makes it impossible to use a non-standard log level and a proxy at the same time :) Only one of them can be the first item in the list.